subprojects/parseq-restli-client/src/main/java/com/linkedin/restli/client/ParSeqRestClient.java

```diff
@@ -236,7 +213,7 @@ private boolean hasRequestContextTimeout(RequestContext requestContext) {
 
   // check whether we need to apply timeout to a rest.li request task.
   private boolean needApplyTaskTimeout(RequestContext requestContext, ConfigValue<Long> timeout) {
-    // if no timeout configured or per-request timeout already specified in request context
+    // **return false** if no timeout configured or per-request timeout already specified in request context
```

Comment:
**return false** -> return false

I used that formatter just to show you what to be changed.

Reply:
:) I'll fix it